packages/components/src/components/@shared/_kol-link-mixin.scss

```diff
@@ -0,0 +1,36 @@
+@import 'mixins';
+@import '../tooltip/style';
+
+@mixin kol-link-styles($block-classname) {
+	@layer kol-component {
+		:host {
```

Comment:
Gibt's hier einen Grund, dass die Styles in :host und nicht im Block sind, oder könnten wir sie verschieben?

Reply:
Wenn es nicht im :host ist, unterscheidet sich das Design.
![image](https://github.com/user-attachments/assets/ed1d2cde-c654-446e-b1e0-d6f4177ada7a)

Develop:
![image](https://github.com/user-attachments/assets/1a42a4dc-be9c-4ab0-bf8c-8e0bd90009b4)